.github/workflows/tests.yml

```diff
@@ -1,22 +1,22 @@
 name: tzdata contents
 
-on: [push]
+on: [pull_request]
```

Comment:
You probably still want to keep `push`, right?

Reply:
For what purpose? I don't want people pushing directly to `master`, and everything else should go through the PR workflow.

Reply:
I mean, sure. You probably want to setup branch rules then.